Automattic/wp-calypso
# Jetpack Pricing: Adjust layout to account for VideoPress.

#### Changes proposed in this Pull Request

This PR adjusts the layout of the Jetpack pricing page to account for the addition of VideoPress.

![image](https://user-images.githubusercontent.com/42627630/139122671-0461982b-6a87-454c-bff8-106b27df81ed.png)

#### Testing instructions



In Jetpack Cloud:
1. Visit `jetpack.cloud.localhost:3001/pricing?flags=jetpack/only-realtime-products` and view the page at varying widths, verifying that the layout adjusts appropriately.
2. Visit `jetpack.cloud.localhost:3001/pricing`, verifying that the pricing page has not changed.

In Calypso:
1. Visit `calypso.localhost:3000/plans/:site?flags=jetpack/only-realtime-products` and view the page at varying widths, verifying that the layout adjusts appropriately.
2. Visit `http://calypso.localhost:3000/plans/agreeable-nurse.jurassic.ninja`, and verify that the pricing page has not changed.



Related to #1200412004370260-as-1201245515881797
